src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -77,9 +75,13 @@ AppHost::AppHost() noexcept :
     _window->MouseScrolled({ this, &AppHost::_WindowMouseWheeled });
     _window->WindowActivated({ this, &AppHost::_WindowActivated });
     _window->HotkeyPressed({ this, &AppHost::_GlobalHotkeyPressed });
+    _window->NotifyTrayIconPressed({ this, &AppHost::_HandleTrayIconPressed });
     _window->SetAlwaysOnTop(_logic.GetInitialAlwaysOnTop());
     _window->MakeWindow();
 
+    // Update our own internal state tracking if we're in quake mode or not.
+    _IsQuakeWindowChanged(nullptr, nullptr);

```

Comment:
@zadjii-msft I moved this down after `MakeWindow` because it'll need the HWND to create the tray icon, lmk if bad

Reply:
Huh. Does the window appear initially at the default size, then immediately snap to the _quake sizing? IIRC in the CreateWindow path we create the window at the size and position of the `_quake` window. I'm worried that moving this later in the startup, we're going to have the _tiniest_ visual glitch

Reply:
Not that I can tell, but I'll just put it back where it originally was to be safe, then I can just call UpdateTrayIcon here instead